cmd/juju/application/deployer/deployer.go

```diff
@@ -551,9 +551,17 @@ func (d *factory) newDeployBundle(_ charm.Schema, ds charm.BundleDataSource) dep
 }
 
 func (dk *localPreDeployerKind) CreateDeployer(_ context.Context, d factory) (Deployer, error) {
+	deployCharm := d.newDeployCharm()
+	if deployCharm.baseFlag.Empty() {
+		return nil, errors.Errorf("must provide a base with the `--base` flag when deploying pre-deployed local charms")
```

Comment:
Is there another description besides `pre-deployed` we can use here?  As far as I know, `pre-deployed` is a term used in the implementation and no where else.

Reply:
Good point, some more clarity would probably be useful here.